code/modules/jobs/job_types/adventurer/types/combat/sorceress.dm

```diff
@@ -30,10 +30,13 @@
 		H.adjust_skillrank(/datum/skill/craft/alchemy, 2, TRUE)
 		H.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
 		if(H.age == AGE_OLD)
```

Comment:
you are missing the +1 INT when old .. 

also ngl i think its fine as is. if they were to be the same then why dont just merge them